### PR TITLE
Partnerships standard ops

### PR DESCRIPTION
**Pull request by colliand**

### CODE REVIEW

File: partnerships/workflow.md [hunk 1]
```diff
@@ -1,8 +1,94 @@
 # Workflow
 
-We are still working out the workflow of the Partnerships team at 2i2c.
-Check out these issues for some conversations around this:
+% We are still working out the workflow of the Partnerships team at 2i2c.
+% Check out these issues for some conversations around this:
+
+% - [System for planning and coordination](https://github.com/2i2c-org/team-compass/issues/510)
+% - [System for leads and contacts](https://github.com/2i2c-org/leads/issues/99)
+% - [System for monthly invoicing](https://github.com/2i2c-org/team-compass/issues/355)
```
Comment (consideRatio) [inline partnerships/workflow.md:8]: Oh today I learned these are MyST comments: https://myst-tools.org/docs/mystjs/blocks#comments

**Comment by jmunroe** [comment]
@colliand and Camille met on 2023-02-06 to discuss SOPs on Lead->Prospect->Partner

@colliand explained the difference between lead/prospect/partner.

Notes: 

SOP: Hail the Lead
> Task to be initiated by an email to partnerships@2i2c.org 
> A SOP needs to start with information on how to get on to FreshDesk. May seem basic but the intended audience needs to be someone is completely new to the procedure.
> The intended outcome is the scheduling of a Calendly meeting

SOP: Prep for First Meeting with Lead
> The request for a Calendly meeting starts this process.
> Create folder in Google Drive for lead
> We need explore if Calendly the correct platform to automate this process.
> Decision: Does the lead verbally agree to become a prospect?
> Outcome: if no, move the Google Folder
> Outcome: if yes, prepare and a send a draft contract
> Send resolution as ticket to FreshDesk

SOP: Send Draft Contract / Negotiate Agreement
> Only certain files within the Google folder are to be shared (running notes and draft service agreement)
> Make explicit that all other files in that folder are not shared.

SOP: DocuSign Agreement
> Send to CS&S for signature (needs meeting with CS&S to work at the details)

We will need follow up on the Partnership phase (e.g. launch meeting, invoicing, satisfication check-in, renewal tickler system) to be sketched out by @jmunroe 

Question: Will we be able to collect the required sales analytics from this data collection?
Answer: We believe so assuming the number of leads/prospects is < 1/day. 

This new set of SOPs puts the responsibility on @2i2c-org/partnerships-and-community-guidance to inform @2i2c-org/engineering of upcoming new hub deployments. 

Question: How long does it take to go from Prospect -> Deployment on the engineering side?
Answer: at least 2 weeks, but realistically 3-4 weeks.

Is there new software besides FreshDesk/Calendly/GoogleDrive that we should be considering? 
Perhaps SmartSheet is something to consider in the future.

**Comment by colliand** [comment]
I request a review (and merge if approved) of this pull request by @jmunroe . This PR provides (first versions of) standard operating procedures for managing leads --> prospects --> partnership. We need to carry out additional work to manage interactions with ongoing partnerships around renewals and satisfaction check-ins. Additional work is also required to extract data to measure 2i2c's leads --> prospects --> partnerships performance. I thank @jmunroe and 2i2c's virtual assistant Camille Gonzalez for meeting with me earlier today and additional thanks to James for capturing important questions in a comment above.

**Review by jmunroe — APPROVED** [review]
I'm happy to merge this as we continue to iterate on our processes.

**Pull request merged**